packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -70,5 +70,9 @@ export const messages = {
   force_lightning_app_create_text: 'SFDX: Create Lightning App',
   force_lightning_component_create_text: 'SFDX: Create Lightning Component',
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
-  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface'
+  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
+  warning_prompt_file_override:
+    'File(s) with specified path already exist in workspace. Would you like to override?',
```

Comment:
Should "override" be "overwrite"?

Reply:
Right, good catch. Just started to notice that I've been mentally using them interchangeably which is incorrect. I'll update.